apps/browser/src/platform/popup/services/browser-file-download.service.ts

```diff
@@ -41,4 +28,31 @@ export class BrowserFileDownloadService implements FileDownloadService {
       window.document.body.removeChild(a);
     }
   }
+
+  private async downloadSafari(
+    request: FileDownloadRequest,
+    builder: FileDownloadBuilder,
+  ): Promise<void> {
+    let data: BlobPart = null;
```

Comment:
suggestion: it seems you are explicitly converting the data to `string`
```suggestion
    let data: string = null;
```